src/Cli/dotnet/NugetPackageDownloader/NuGetPackageDownloader.cs

```diff
@@ -450,10 +450,9 @@ private IEnumerable<PackageSource> LoadNuGetSources(PackageId packageId, Package
                 throw new NuGetPackageNotFoundException(
                     string.Format(
                         LocalizableStrings.IsNotFoundInNuGetFeeds,
-                        $"{packageIdentifier}::{versionRange}",
+                        $"{packageIdentifier} of version range {versionRange}",
```

Comment:
I wonder if this expression make it clearer for customers

Reply:
I might suggest changing the formatting a bit based on the kind of range:
* if the range is exact match, say `Version {versionRange} of package {packageIdentifier}`
* if the range is 'greater than or equal to', say `A version between {min} and {max} of package {packageIdentifier}`
  * this works for 'less than or equal to' as well
* if the range is 'at least', say `A version higher than {minBound} of package {packageIdentifier}`
* if the range is 'at most', say `A version less than than {maxBound} of package {packageIdentifier}`